tests/foreman/cli/test_variables.py

```diff
@@ -16,122 +16,185 @@
 @Upstream: No
 """
 
+from robottelo import ssh
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.cli.environment import Environment
+from robottelo.cli.factory import make_hostgroup, make_variable
+from robottelo.cli.host import Host
+from robottelo.cli.hostgroup import HostGroup
+from robottelo.cli.proxy import Proxy
+from robottelo.cli.puppet import Puppet
+from robottelo.cli.variable import SmartVariable
```

Comment:
Renamed module name, test module name left as it was as we need to re-name also similar modules for UI and API, should be performed in scope of renaming task